# stack information for a single running process

uint8 MAX_REPORT_TASK_NAME_LEN = 16

uint16 stack_free
uint8[16] task_name
